### ISSUE TITLE
Improve snapshots command to allow object storage repository

### ISSUE THREAD

**Issue by rolljee** [kuzzleio/kourou#129]
Current feature only allow fs snapshot, this should be improved to allow other ways of storing snapshots

**Comment by GuinierRemi** [comment]
@rolljee is this still needed ?

**Comment by rolljee** [comment]
Not really, our use cases around this only needed fs storage so far. This could be a good feature to have, or it might be needed in the futur, but for now this is not a priority at all
